docs/community/governance.md

```diff
@@ -0,0 +1,180 @@
+
+* **The Observatorium project**: The sum of all activities performed under the [Observatorium organization on GitHub][gh], concerning one or more repositories or the community.
+
+Both Triage and Maintainers have are part of [`observatorium-team@googlegroups.com`][team] email list.
+
+## Values
+
+The Observatorium developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc].
+Furthermore, the Observatorium community strives for kindness, giving feedback effectively, and building a welcoming environment. The Observatorium developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Decision making
+
+### Maintainers Team
+
+Team member status may be given to those who have made ongoing contributions to the Observatorium project for at least 3 months.
+This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account.
+
+New members may be proposed by any existing Maintainer by email to [Observatorium][team]. It is highly desirable to reach consensus about acceptance of a new member.
+However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote) of Team Maintainers.
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership.
+This email will also be CC'd to [Observatorium][team] for record-keeping purposes.
+
+If they choose to accept, the following steps are taken:
+
+* Maintainer is added to the [GitHub organization][gh] as _Owner_.
+* Maintainer is added to the [Observatoriumo][team].
+* Maintainer is added to the list of team members [here][maintainers-doc]
+* New maintainer is announced on the [Observatorium Twitter](https://twitter.com/0bservatorium) by an existing team member.
+
+Team members may retire at any time by emailing [observatorium-team@googlegroups.com][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [observatorium-team@googlegroups.com][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+### Technical decisions
+
+Smaller technical decisions are made informally and [lazy consensus](#consensus) is assumed. Technical decisions that span multiple parts of the Observatorium project
+should be discussed and made on the [GitHub issues][issues] and in most cases followed by proposal as described [here](/CONTRIBUTING.md#adding-new-features--components).
+
+Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
```

Comment:
We already discussed lazy consensus earlier

Reply:
we have it also on Prom/Thanos so I would stick to this, some repetition maybe is fine 🙈